ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh

```diff
@@ -134,7 +134,17 @@ fi
 arch_instance_type=$(echo -n "${CONTROL_PLANE_INSTANCE_TYPE}" | cut -d . -f 1)
 BOOTSTRAP_NODE_TYPE=${arch_instance_type}.large
 
-workers=3
+if [[ -n "${COMPUTE_NODE_REPLICAS}" ]]; then
+  if [[ "${COMPUTE_NODE_REPLICAS}" -le 0 ]]; then
+    workers=0
+  elif [[ "${COMPUTE_NODE_REPLICAS}" -gt 3 ]]; then
+    workers=3
+  else
+    workers="${COMPUTE_NODE_REPLICAS}"
+  fi
+else
+  workers=3
+fi
```

Comment:
Maybe we can simplify this to:
```suggestion
workers=${COMPUTE_NODE_REPLICAS:-3}
if [[ "${COMPUTE_NODE_REPLICAS}" -le 0 ]]; then
    workers=0
elif [[ "${COMPUTE_NODE_REPLICAS}" -gt 3 ]]; then
    workers=3
fi
```

Reply:
of course, done.

Do you think it is worth to validate in the script that the value of `$COMPUTE_NODE_REPLICAS` is a whole number? Or just let it fail later on otherwise?